README.md

```diff
@@ -69,7 +69,7 @@ NetSuite.configure do
   api_version	'2012_1'
 
   # optionally specify full wsdl URL (to switch to sandbox, for example)
-  wsdl          "https://webservices.sandbox.netsuite.com/wsdl/v#{api_version}_0/netsuite.wsdl"
+  wsdl          "https://system.netsuite.com/wsdl/v#{api_version}_0/netsuite.wsdl"
```

Comment:
should this be `"https://webservices.netsuite.com/wsdl/v#{api_version}_0/netsuite.wsdl"`? In order to get this working correctly for our current setup we had to use the former url not the `system` one

Reply:
Correct.  You may have to add in the Data Center (ie. na1, na2) if users aren't going to use data center agnostic rules. 

From NetSuite Help Docs:

The format of an account-specific domain name is: <account ID>.<service>.netsuite.com. For example, if your account ID is 123456, your account-specific domains would be:
     123456.suitetalk.api.netsuite.com
     123456.restlets.api.netsuite.com
     123456.shop.netsuite.com

As of 2017.2, account-specific domains are supported for SuiteTalk (web services), RESTlets, and External Catalog (WSDK) shopping sites. These account-specific domains are unique to your account because they contain your account ID as part of the domain name. These domains do not change when your account is moved to a different data center.

Format of a Data Center-Specific NetSuite URL

For web services, rest, and External Catalog (WSDK) site URLs, consider using account-specific URLs rather than data center-specific URLs. See URLs for Account-Specific Domains.

Data center-specific NetSuite URLs are strings made up of the following parameters: <ServiceType>.<LocationId>.<AccountType>.netsuite.com.

Reply:
Ignorant question: How do you find your account id?

Reply:
@cooncesean Navigate to the SuiteTalk preferences via “Setup > Integration > Web Services Preferences” to find this ID.